internal/comit_directories/src/lib.rs

```diff
@@ -0,0 +1,20 @@
+use directories::ProjectDirs;
+use std::path::PathBuf;
+
+// Linux: /home/<user>/.config/<name>/
```

Comment:
Any chance you'd be keen to do #1259 at the same time?

Reply:
Just got made aware of this comment now, will do.